server/middleware/setUpSentry.ts

```diff
@@ -1,24 +1,38 @@
 import * as Sentry from '@sentry/node'
-import '@sentry/tracing'
+import * as Tracing from '@sentry/tracing'
 import express from 'express'
 import applicationVersion from '../applicationVersion'
 import config from '../config'
 
 export function setUpSentryRequestHandler(app: express.Express): void {
   if (config.sentry.dsn) {
+    // Prevent usernames which are PII from being sent to Sentry
+    // https://docs.sentry.io/platforms/python/guides/logging/data-management/sensitive-data/#examples
+    const anonymousID = Math.random().toString()
```

Comment:
We're generally using `roomId`, `premisesId` etc., would change to `anonymousId` for consistency

Reply:
Didn't spot that, will change! 👍